examples/plugin/hide-internal-api/build.gradle.kts

```diff
@@ -30,8 +32,8 @@ dependencies {
 val dokkaOutputDir = "$buildDir/dokka"
 
 tasks {
-    withType<KotlinCompile> {
-        kotlinOptions.jvmTarget = "1.8"
+    withType<KotlinCompile>().configureEach {
+        (compilerOptions as? KotlinJvmCompilerOptions)?.jvmTarget?.set(JvmTarget.JVM_1_8)
```

Comment:
You could also use `KotlinJvmCompile`

Reply:
3ad37d58688831e90c3b9bdd49078af920a02c9a